gapis/api/vulkan/frame_loop.go

```diff
@@ -587,6 +593,23 @@ func (f *frameLoop) buildStartEndStates(ctx context.Context, startState *api.Glo
 				f.pipelineLayoutToCreate[pipelineLayout] = true
 			}
 
+		// PipelineCache(s)
+		case *VkCreatePipelineCache:
+			vkCmd := cmd.(*VkCreatePipelineCache)
+			pipelineCache := vkCmd.PPipelineCache().MustRead(ctx, vkCmd, currentState, nil)
```

Comment:
PPipelineCache is a typo, right? Does this build?

Reply:
func (c *VkCreatePipelineCache) PPipelineCache() VkPipelineCacheᵖ { ...

Sadly not!